### DISCUSSION TITLE
Portuguese translations

### DISCUSSION THREAD

**Discussion by gpsilv4** [opendatateam/udata#2791] [Q&A]
I would like to add the Portuguese translations in udata-front, how do I do it?

**Comment by maudetes** [comment]
Hello !

I've added the Portuguese language in the [udata-front crowdin project](https://crowdin.com/project/udata-front).
The process is the same for udata-front as udata. You can take a look at this documentation if needed: https://udata.readthedocs.io/en/latest/adding-translations/.

You should then update your udata-front version with the release with the latest translations shipped when merged and released.
